sentry_sdk/integrations/gcp.py

```diff
@@ -31,13 +34,13 @@
 
 def _wrap_func(func):
     # type: (F) -> F
-    def sentry_func(*args, **kwargs):
-        # type: (*Any, **Any) -> Any
+    def sentry_func(functionhandler, event, *args, **kwargs):
+        # type: (Any, Any, *Any, **Any) -> Any
```

Comment:
```suggestion
    def sentry_func(*args, **kwargs):
        # type: (*Any, **Any) -> Any
```

Just to keep the diff small, this change was not necessary